CRM/Contact/BAO/Query.php

```diff
@@ -4705,7 +4708,7 @@ public static function appendAnyValueToSelect($selectClauses, $groupBy) {
    *
    */
   public static function getGroupByFromOrderBy(&$groupBy, $orderBys) {
-    if (CRM_Utils_SQL::disableFullGroupByMode()) {
+    if (!CRM_Utils_SQL::disableFullGroupByMode()) {
```

Comment:
This is a bug which went unnoticed till now, reason why this function is acting odd unlike other fn which handle FULL_GROUP_BY query fragments.